### PR TITLE
several changes to the scripts

### PR DESCRIPTION
**Pull request by richieam93**

### CODE REVIEW

**Comment by richieam93** [comment]
Hello Phoenix 49
is that OK?
do you need anything else?

File: custom_components/huggingchat_conversation/translations/de.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "step": {
+      "init": {
+        "data": {
+          "chat_model": "Chat-Modell",
+          "prompt": "Prompt-Vorlage",
+          "enable_assistants": "Assistenten aktivieren",
+          "assistant_name": "Assistenten-Name",
+          "web_search": "Web-Suche aktivieren",
+          "web_search_engine": "Suchmaschine",
+          "web_search_prompt": "Suchanfrage-Vorlage"
```
Comment (PhoenixR49) [inline custom_components/huggingchat_conversation/translations/de.json:25]: J'ai vérifié sur un traducteur et la traduction de ce que je veux est "Aufforderung zur Suchanfrage". Dites moi si ce

Comment (PhoenixR49) [inline custom_components/huggingchat_conversation/translations/de.json:25]: I checked on a translator and the translation of what I want is "Aufforderung zur Suchanfrage"

**Review by PhoenixR49 — CHANGES_REQUESTED** [review]
All changes are correct except for “web_search_prompt” (line 25)

**Review by PhoenixR49 — CHANGES_REQUESTED** [review]
I'd like to keep the `duckduckgo-search` package (version 6.3.3)

**Comment by richieam93** [comment]
Ich habe alle Dateien nochmal angepasst. Mit der Version duckduckgo-search==3.9.11 hat es geklappt.

Ich werde mal prüfen, ob es auch mit der Version 6.3.3 klappt.

**Comment by richieam93** [comment]
version 6.3.3 now also works :)

**Review by richieam93 — COMMENTED** [review]
this is how everything should work

**Comment by PhoenixR49** [comment]
Thanks for your work, the changes have been added to the repository.

**Pull request closed without merging**